### DISCUSSION TITLE
SO 2.3.100 Elasticsearch error

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#7200] [Unsupported Versions] [answered]
Hello!

Since I have updated to 2.3.100 I get the following Elasticsearch error in Kibana:

![image](https://user-images.githubusercontent.com/81571847/154017439-02444d4e-2199-4b04-aacd-c85f7fd0dc12.png)

How can I solve that one?

Cheers, Ben

**Comment by palichx** [comment, marked as answer]
Hi,
Try running this on the manager:

salt-call state.apply kibana.so_savedobjects_defaults -l info

**Reply by ben-sec** [reply]
That did the trick!

**Reply by dougburks** [reply]
For the benefit of others who may run across this discussion, this is documented on the soup page:
https://docs.securityonion.net/en/2.3/soup.html#kibana
